### ISSUE TITLE
Notification Like Desktop

### ISSUE THREAD

**Issue by testforpostman7** [MattIPv4/macOSNotifJS#41]
Hello,

   Is there any possibility to display notification like desktop because I have to display this notification like desktop if I am not on current notification screen then also get notification.

Thanks.

**Comment by MattIPv4** [comment]
No, this library is not intended to interact with or spawn actual desktop notifications. The aim is to emulate the style of macOS desktop notifications within a web page.

**Comment by testforpostman7** [comment]
Okay, thanks for your quick reply.
